pandas/core/generic.py

```diff
@@ -10365,7 +10365,7 @@ def any(
         skipna: bool_t = True,
         level: Level | None = None,
         **kwargs,
-    ) -> Series | bool_t:
+    ):
```

Comment:
Can you keep the return type here? `-> DataFrame | Series | bool_t`?

Also, as this is only used internally, we can make its arguments keyword-only right away without warning

Reply:
By making arguments keyword-only, do you mean placing arguments after `,*,`?